internal/tools/seed.go

```diff
@@ -78,6 +79,7 @@ func SeedBuilds(db *gorm.DB) ([]builds.Build, error) {
 			VersionString: "gcr.io/workspacemanager:1.2.0",
 			CommitSha:     "6a5s4df",
 			BuildURL:      "https://build.3.log",
+			BuiltAt:       time.Now().Add(-6 * time.Hour),
```

Comment:
Assuming the -6 is to put the time into the past, this is fine (but I guess could use a comment). I thought for a sec this was to adjust for timezone but I'm pretty sure that's not the case (cuz it wouldn't work).

Reply:
good call ✅